public/stylesheets/sass/Navbar.scss

```diff
@@ -1,66 +1,177 @@
+	@include size($width: 50%, $height: 100%);
+	@include flex($justify: flex-start,  $align: center);
+	margin-left: 25px;
+
+	@include tablet {
+		@include size($width: 90%, $min-height: 50px);
+		@include flex($justify: space-around);
+	}
+
+	@include mobile {
+		@include size($width: 90%, $min-height: 50px);
+		@include flex($justify: space-around);
+	}
+
+	@include smallMobile {
+		@include size($width: 90%, $min-height: 50px);
+		@include flex($justify: space-around);
 	}
 }
-.NavLinks{
-	@include size($width:50%, $height:100%);
-  @include flex($align: center, $direction: row, $justify: flex-end);
-	list-style: none;
+
+.Hamburger {
+	display: none;
+	@include tablet {
+		@include size($width: 5%);
+		display: block;
+	}
+
+	@include mobile {
+		@include size($width: 5%);
+		display: block;
+	}
+
+	@include smallMobile {
+		@include size($width: 5%);
+		display: block;
+	}
+}
+
+.LogoImage {
+	@include tablet {
+		@include size($width: 95%);
+		@include flex($justify: center);
+	}
+
+	@include mobile {
+		@include size($width: 95%);
+		@include flex($justify: center);
+	}
+
+	@include smallMobile {
+		@include size($width: 95%);
+		@include flex($justify: center);
+	}
+}
+
+.NavLinks {
+	@include size($width: 50%, $min-height: 100%);
+	@include flex($justify:flex-end, $align:center, $direction: row);
+	list-style: flex;
 	margin: 0;
-	padding-right: 10%;
-	list-style: none;
 	color: $white-color;
 
 	@include tablet {
-		@include flex($justify: space-around);
-		padding-right: 5%;
+		@include size($width: 90%, $min-height: 20vh);
+		display: none;
+		justify-content: center;
+		flex-direction: column;
+		font-size: 1.5em;
+	}
+
+	@include mobile {
+		@include size($width: 90%, $min-height: 20vh);
+		display: none;
+		flex-direction: column;
+		justify-content: center;
+		font-size: 1.5em;
+	}
+
+	@include smallMobile {
+		@include size($width: 100%, $min-height: 20vh);
+		display: none;
+		flex-direction: column;
+		justify-content: center;
+		font-size: 1.5em;
+		padding: 0;
 	}
 }
+
 .Link{
 	@include size($width:24%, $height:60%);
 	@include flex($align: center, $justify: center);
 	text-decoration: none;
 	flex-wrap: wrap;
 	color: $white-color;
+	list-style: none;
+
 
+	@include tablet {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+	}
+
+	@include mobile {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+	}
+
+	@include smallMobile {
+		@include size($width: 90%, $height: 10vh);
+		border-top: 1px solid rgba(0,0,0,0.4);
+		font-size: 1em;
+	}
 }
-.Link:hover{
+
+.Link:hover {
 	background-color: rgba(0,0,0,0.7);
 	text-decoration: none;
-	color: #34495e;
+	color: rgba(52, 73, 94,1);
 	font-size: 1.1em;
 }
-.Link:hover a{
+
+.Link:hover a {
 	background-color: rgba(0,0,0,0.7);
 	color: $white-font-color;
 	font-size: 1.1em;
 }
 
-.NavLinks a{
+.NavLinks a {
 	text-decoration: none;
 	font-size: 1.1em;
 	color: $white-font-color;
 	text-align: center;
```

Comment:
Utilities devem ser utilizadas quando for necessário aplicar um estilo diferente daquele definido para o componente.